base_global_planner: "waypoint_global_planner/WaypointGlobalPlanner"
planner_frequency: 0.0
planner_patience: 1.0

WaypointGlobalPlanner:
  max_vel_x: 0.4
  max_vel_theta: 0.3
  inflation_radius: 0.3
  obstacle_weight: 10.0
  resolution: 0.05
  waypoint_tolerance: 0.2
  lookahead_distance: 1.0
